StRoot/StTriggerUtilities/Eemc/EEdsm1.h

```diff
@@ -3,14 +3,11 @@
 /**************************************************************
  * $Id: EEdsm1.h,v 1.2 2012/12/12 22:54:03 fisyak Exp $
  **************************************************************/
-#ifdef __APPLE__
-#include <sys/types.h>
-#endif
```

Comment:
Thank you for your review. The reason why the include is "not needed" on non-apple here is because the stdlib.h shipped with glibc happens to #include sys/types.h under the hood (by an accidental reason). This behavior is glibc-specific and likely will not work with another libc. The sys/types.h include should have been unconditional here, this works only by accident.

The reason why including stdlib.h will work with cint is because it provides its own stub for it (see tcint/cint/include/stdlib.h) that doesn't include sys/types.h.

I am removing use of non-standard types coming from sys/types.h from entire codebase to ensure that even if people have not missed the failing sys/types.h include, we would still be able to remove it with confidence.

The practical problem here is that CINT is not able to work with modern libc's, as it's not compliant and outdated. There are alternative approaches we could take here, but that would require recompiling the ROOT5, or switching to ROOT6 entirely.